llvm/include/llvm/IR/DataLayout.h

```diff
@@ -299,6 +302,17 @@ class DataLayout {
     return ManglingMode == MM_WinCOFFX86;
   }
 
+  uint64_t getNonZeroValueForAddrSpace(uint64_t AddrSpace) {
+  auto It = AddrSpaceToNonZeroValueMap.find(AddrSpace);
+  if (It == AddrSpaceToNonZeroValueMap.end())
+    return 0x000000000;
+  return AddrSpaceToNonZeroValueMap[AddrSpace];
```

Comment:
Rename this function to `getNullPointerValue` - whether that value is non-zero doesn't matter, the goal is that users should query the DL instead of assuming null == 0.
```suggestion
  uint64_t getNonZeroValueForAddrSpace(uint64_t AddrSpace) {
  auto It = AddrSpaceToNonZeroValueMap.find(AddrSpace);
  if (It == AddrSpaceToNonZeroValueMap.end())
    return 0;
  return It->second;
```

Reply:
renamed to getNullPointerValue.